pages/release.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Release Archive
+permalink: "/release/"
+redirect_from: 
+  - "/release_notes/"
+  - "/release_notes.html"
+---
+
+<div>
+<hr>
+<h3>2016</h3>
```

Comment:
liquid may have an operation to get the first item in a list.   Maybe could use this with `site.categories.release` to get the year.